### ISSUE TITLE
Has this incubation stalled?

### ISSUE THREAD

**Issue by marcoscaceres** [dway123/raw-clipboard-access#8]
There appears to have been very little activity with this repository in the last few years. 

Is it ok to archive this repo? 

Unless anyone objects, we will go ahead and archive it in a week or so.

**Comment by inexorabletash** [comment]
Yes, this path was abandoned and effectively replaced with "web custom format" support:

* https://github.com/w3c/editing/blob/gh-pages/docs/clipboard-pickling/explainer.md
* https://github.com/w3c/clipboard-apis/pull/175

I think we're good to archive. Maybe leave breadcrumbs to the above in the README?

**Comment by marcoscaceres** [comment]
Ok, cool. Thanks @inexorabletash. Will do!
